examples/long-running/long-running-script.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use DDTrace\SpanData;
+
+// This part is required for long running processes
+\DDTrace\trace_function('repetitive_function', function (SpanData $span, $args) {
+    $span->service = 'my_service';
+    // Optional
+    $span->meta['count'] = $args[0];
```

Comment:
Do we need this at all? Unless there's something I'm missing, I don't see the value in it and users may just copy/paste and leave it in there for no reason.